Client/src/Components/NavBar/index.jsx

```diff
@@ -103,22 +105,11 @@ function NavBar() {
     >
       <Container maxWidth="xxl" sx={{ width: "100%" }}>
         <Toolbar disableGutters>
-          <Typography
-            variant="h6"
-            noWrap
-            component="a"
-            href="#"
-            sx={{
-              mr: 2,
-              display: { xs: "none", md: "flex" },
-              fontWeight: theme.typography.fontWeightBold,
-              letterSpacing: theme.spacing(0.3),
-              color: "var(--env-var-color-11)",
-              textDecoration: "none",
-            }}
-          >
-            Peak Watch
-          </Typography>
+          <img
+            id="bw-uptime-logo-dashboard"
+            src={BWULogo}
+            alt="BlueWave Uptime Logo"
+          />
```

Comment:
Please use svg instead of a regular img. Svgs are faster to load, scale better and you can also use inline css styles to edit it. Check `navBar` component to see implementation.